src/js/controllers/ctrl-media-import.js

```diff
@@ -184,6 +206,20 @@ app.controller('MediaImportCtrl', function ($scope, $window, $timeout) {
 		})
 	}
 
+	const deleteAsset = (media) => {
+		IS_HIDDEN_CLICK = true
+
+		// if is deleted update the local and server version.
+		if (media.is_deleted == 0) {
+			AssetSrv.deleteAsset(media)
+			media.is_deleted = '1'
+		}
+		else {
+			AssetSrv.restoreAsset(media)
+			media.is_deleted = '0'
+		}
```

Comment:
The potential value of `media.is_deleted` seems inconsistent. If the value of `is_deleted` is meant to be boolean, prefer `false` over '0' and `true` over `'1'`. If all you care about is whether `is_deleted` is falsy, use `! media.is_deleted`.